src/orion/core/utils/tests.py

```diff
@@ -37,6 +40,74 @@ def default_datetime():
     return datetime.datetime(1903, 4, 25, 0, 0, 0)
 
 
+def generate_trials(trial_config, statuses):
+    """Generate Trials with different configurations"""
+
+    def _generate(obj, *args, value):
+        if obj is None:
+            return None
+
+        obj = copy.deepcopy(obj)
+        data = obj
+
+        for arg in args[:-1]:
+            data = data[arg]
+
+        data[args[-1]] = value
+        return obj
+
+    new_trials = [_generate(trial_config, 'status', value=s) for s in statuses]
+
+    for i, trial in enumerate(new_trials):
+        trial['submit_time'] = datetime.datetime.utcnow() + datetime.timedelta(seconds=i)
+        if trial['status'] != 'new':
+            trial['start_time'] = datetime.datetime.utcnow() + datetime.timedelta(seconds=i)
+
+    for i, trial in enumerate(new_trials):
+        if trial['status'] == 'completed':
+            trial['end_time'] = datetime.datetime.utcnow() + datetime.timedelta(seconds=i)
+
+    # make each trial unique
+    for i, trial in enumerate(new_trials):
+        if trial['status'] == 'completed':
+            trial['results'].append({
+                'name': 'loss',
+                'type': 'objective',
+                'value': i})
+
+        trial['params'].append({
+            'name': 'x',
+            'type': 'real',
+            'value': i
+        })
+
+    return new_trials
+
+
+@contextmanager
+def create_experiment(exp_config=None, trial_config=None, statuses=None):
+    """Context manager for the creation of an ExperimentClient and storage init"""
+
+    if exp_config is None:
+        raise ValueError("Parameter 'exp_config' is missing")
+    if trial_config is None:
+        raise ValueError("Parameter 'trial_config' is missing")
+    if statuses is None:
+        statuses = ['new', 'interrupted', 'suspended', 'reserved', 'completed']
+
+    from orion.client.experiment import ExperimentClient
```

Comment:
Can we move this import to top module?

Reply:
Unfortunately no, it creates a circular dependency otherwise because `Experiment` has a hard dependency on the storage.